tasks/src/main/java/io/MainTask.java

```diff
@@ -0,0 +1,106 @@
+package io;
+
+import static java.nio.file.FileVisitResult.CONTINUE;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.FileVisitResult;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.SimpleFileVisitor;
+import java.nio.file.attribute.BasicFileAttributes;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+class FileTreeVisitor extends SimpleFileVisitor<Path> {
+
+  static String fileVisitResult = "";
```

Comment:
That could be non-static. Use an instance of the FileTreeVisitor class in the getFileTree method